.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run test && npm run test:ct && npm run lint
```

Comment:
Woah, do you really want to run component tests too? :-D folks will be crazy waiting for tests to complete. better just leave it on per-pull-request CI?

Reply:
I think everyone runs component tests before pushing to CI. Usually what I do is run the tests to check if tests are passing, if not update tests&snapshots and then push to master.

Reply:
With this, you will be sure that your PR does not fail tests&lint once and always after  you create a PR.

Reply:
Also you can easly disable husky while creating a WIP commit with **-n** or **--no-verify**

Reply:
ok, I see, still sometimes you'd need to commit quickly, but the flag should help. lgtm 👍🏼